server/public/templates/admin/settings.html

```diff
@@ -35,6 +35,31 @@ <h5>Global</h5>
                 <span id="helpBlock" class="help-block">Secret key for cookie sessions, should be changed before deploying the application. Changing this will invalidate existing sessions.</span>
               </div>
             </div>
+            <div class="form-group">
+              <label for="global.FrontPopupEnable" class="col-xs-4 col-sm-3 control-label">Enable One Time Modal</label>
+              <div class="col-xs-8 col-sm-9">
+                <div class="checkbox">
+                  <label>
+                    <input type="checkbox" name="global.FrontPopupEnable" ng-model="settings.global.FrontPopupEnable"> Enable showing a modal to new visitors. This can be used for showing a disclamer or Site/Conact information. 
```

Comment:
```suggestion
                    <input type="checkbox" name="global.FrontPopupEnable" ng-model="settings.global.FrontPopupEnable"> Enable showing a modal to new visitors. This can be used for showing a disclamer or Site/Contact information. 
```

typo :)